lib/runner.ex

```diff
@@ -30,14 +42,16 @@ defmodule Runner do
     modules
     |> Enum.drop_while( &(&1 != start_module))
     |> Enum.take_while( &(run_module(&1) == :passed))
+    |> done?
   end
 
+  defp done?(modules), do: modules == @modules
```

Comment:
What's the desired behavior when you override the initial koan? In that case would it only run from that one to the last one?